packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -253,15 +253,15 @@ export class EditorToolbarComponent {
 	/* Publish and unpublish */
 
 	/**
-	 * Returns the text present for the save/publish button.
+	 * Waits for the save/publish button.
 	 *
 	 * @returns {Promise<string>} String found on the button.
 	 */
-	async getPublishButtonText(): Promise< string > {
```

Comment:
Oh, it looks like we indeed didn't use this method before, did we?

Reply:
Nope. I think I stopped using it a while ago, and forgot to clean up.